compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1366,16 +1366,9 @@ pub enum NullOp<'tcx> {
     AlignOf,
     /// Returns the offset of a field
     OffsetOf(&'tcx List<(VariantIdx, FieldIdx)>),
-    /// Returns whether we want to check for library UB or language UB at monomorphization time.
-    /// Both kinds of UB evaluate to `true` in codegen, and only library UB evalutes to `true` in
-    /// const-eval/Miri, because the interpreter has its own better checks for language UB.
-    UbCheck(UbKind),
-}
-
-#[derive(Clone, Copy, Debug, PartialEq, Eq, TyEncodable, TyDecodable, Hash, HashStable)]
-pub enum UbKind {
-    LanguageUb,
-    LibraryUb,
+    /// Returns whether we want to check for UB at monomorphization time.
+    /// This returns the value of `cfg!(debug_assertions)`.
```

Comment:
Elsewhere we mention that this is `cfg!(debug_assertions)` _at monomorphization time_, to indicate that this is not the cfg at expansion time. Should we mention "at monomorphization time" here as well? If you think it's too redundant, keeping this the way it is would be fine with me.

Reply:
Sure, I can add it here. We'll have to remember to change it again then if we want to give https://github.com/rust-lang/rust/pull/122282 another shot.